src/main/sh/mommy

```diff
@@ -350,6 +350,7 @@ while getopts ":hv1d:c:e:s:-:" OPT; do
     c|config) opt_config="$OPTARG" ;;
     e|eval) require_arg; opt_eval="$OPTARG" ;;
     s|status) require_arg; require_int; opt_status="$OPTARG" ;;
+    r|rename) require_arg; opt_rename="$OPTARG" ;;
```

Comment:
since renaming is a miscellaneous option that is unlikely to be used very often, i think we shouldn't add a short option, only a long option. so keep `--rename` as is, but could you please remove `-r`? (also from the readme, of course)